readme.md

```diff
@@ -247,88 +252,47 @@ var handleClickEvent = function(){
 button.on("click", handleClickEvent);
 ```
 
-Try this-- insert the following line anywhere in our click-handler:
-* `console.log($(this))`
-* Now what do you see when we click the button? How would you define `this` in the context of an event listener?
+Insert the following line anywhere in our click-handler...
 
-<details>
-<summary>
-Stumped?
-</summary>
-In the context of an event listener callback, `$(this)` always refers to the object that triggered the event.
-</details>
+```js
+console.log($(this))
+```
 
-### You Do: `this` Practice (3:55 - 4:05, 10 minutes)
-
-Clone and follow the instructions in this [repository](https://github.com/ga-wdi-exercises/events-this-practice).
+Refresh the page. What do you see when you click the button?
 
 <details>
-<summary>
-Bonus: `this` and Color Scheme Picker
-</summary>
-<p>
-If you're curious, here's a short solution to the earlier Color Scheme Picker exercise that makes use of `this`. Keep in mind, this is advanced! You are not expected to be able to write code like this yet.  
-</p>
-`
-var buttons = $("li").on("click", function () {
-  $("body").attr("class", $(this).attr("class"));
-});
-`
-</details>
 
-### Event Defaults (4:05 - 4:10, 5 minutes)
+  <summary><strong>With that in mind, how would you define `this` in the context of an event listener?</strong></summary>
 
-Back in the code we were using in-class, replace your button with a link to Google...
+  In the context of an event listener callback, `$(this)` always refers to the object that triggered the event.
 
-```html
-<body>
-  <a href="http://google.com">Click me!</a>
-</body>
-```
+</details>
 
-Now, add an event listener to that link that brings up a `prompt` box, asking the user if they want to go to Google...
+### You Do: [`this` Practice](https://github.com/ga-wdi-exercises/events-this-practice) (10 minutes / 1:25)
```

Comment:
recommend moving script to bottom of page or be prepared to explain why it's in the head here